src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,28 @@
+const Task = require('../../db/models/task/index');
+
+module.exports.getAllTasks = (req, res, next) => {
+  Task.find().then(result => {
+    res.send({ data: result });
+  });
+};
+
+module.exports.createNewTask = (req, res, next) => {
+  const task = new Task(req.body);
```

Comment:
whats happends if i don't send data ?

Reply:
A new object will be added without data but with the assigned id